regression/scripts/solar_network/solar_network.py

```diff
@@ -61,30 +61,28 @@ def main():
     Power           = 7500.  #Specify either thrust or power to design for
     
     # Design the Propeller
-    prop_attributes = Data()
-    prop_attributes.number_blades       = Blades 
-    prop_attributes.freestream_velocity = Velocity
-    prop_attributes.angular_velocity    = RPM*(2.*np.pi/60.0)
-    prop_attributes.tip_radius          = Radius
-    prop_attributes.hub_radius          = Hub_Radius
-    prop_attributes.design_Cl           = Design_Cl 
-    prop_attributes.design_altitude     = design_altitude
-    prop_attributes.design_thrust       = Thrust
-    prop_attributes.design_power        = Power
-    prop_attributes                     = propeller_design(prop_attributes)
+    prop                 = SUAVE.Components.Energy.Converters.Propeller()
+    prop.number_blades       = Blades 

```

Comment:
is this just a refactoring?  why prop_attributes -> prop?   (just curious)

Reply:
Yes, no more prop attributes. I'm going to correct this and delete this pull request per our discussion this morning